dask/array/tests/test_sparse.py

```diff
@@ -192,3 +192,12 @@ def f():
     x = da.from_delayed(d, shape=(3, 3), meta=sparse.COO.from_numpy(np.eye(1)))
     assert isinstance(x._meta, sparse.COO)
     assert_eq(x, x)
+
+
+def test_from_array():
+    x = sparse.COO.from_numpy(np.eye(10))
+    d = da.from_array(x, chunks=(5, 5))
+
+    assert isinstance(d._meta, sparse.COO)
+    assert_eq(d, d)
```

Comment:
Typo? Should be `x, d, meta=False`?

Reply:
Well observed, I think it should be `assert_eq(x, d)`, but `check_meta=True` (the default), `check_meta=False` is only needed if the Dask array and the array resulting from `.compute()` differ, and the only case I know for that are masked arrays.

Reply:
Submitted PR ( https://github.com/dask/dask/pull/5084 ) to address this. Please take a look 🙂

Reply:
This doesn't really matter that much.  We often use `assert_eq` just to perform a bunch of checks on the provided dask collections.  Here we don't care about equality, we just care about these checks.

Reply:
Maybe we should have a different function that makes this intent clearer?

Reply:
We could.  I don't personally care.  No strong thoughts.  This is a low priority for me.

Reply:
Just suggesting since 3 people were confused about the same thing. 🙂